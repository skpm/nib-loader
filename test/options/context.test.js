/* eslint-disable
  prefer-destructuring,
*/
const webpack = require('../helpers/compiler');

describe('Options', () => {
  describe('context', () => {
    test('{String}', async () => {
      const config = {
        loader: {
          test: /(png|jpg|svg)/,
          options: {
            context: `${__dirname}`,
          },
        },
      };

      const stats = await webpack('fixture.js', config);
      const { source } = stats.toJson().modules[1];

      expect(source).toMatchSnapshot();
    });
  });
});
