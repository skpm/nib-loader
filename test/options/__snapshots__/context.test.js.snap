// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Options context {String} 1`] = `
"/* eslint-disable */
import png from './file.png';

export default png;
"
`;
